src/content/app/species/SpeciesPageContent.tsx

```diff
@@ -119,12 +119,13 @@ const TopBar = () => {
 
   return (
     <div className={styles.topbar}>
-      <div className={styles.topbarLeft}>
-        <span className={styles.pageTitle}>Species Home page</span>
-        <CloseButton
-          className={styles.close}
-          onClick={returnToSpeciesSelector}
+      <div className={styles.topbarLeft} onClick={returnToSpeciesSelector}>
+        <Chevron
+          direction={'left'}
```

Comment:
This can passed as just a string: `direction="left"`

Reply:
Done